lib/features/home/home_page.dart

```diff
@@ -104,38 +120,39 @@ class HomePage extends HookConsumerWidget {
                 ),
               ),
               const SizedBox(height: Grid.xs),
-              Row(
-                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                children: [
-                  Expanded(
-                    child: FilledButton(
-                      onPressed: () {
-                        Navigator.of(context).push(
-                          MaterialPageRoute(
-                            builder: (context) =>
-                                const DepositPage(rfqState: RfqState()),
-                          ),
-                        );
-                      },
-                      child: Text(Loc.of(context).deposit),
+              if (pfis.isNotEmpty)
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    Expanded(
+                      child: FilledButton(
+                        onPressed: () {
+                          Navigator.of(context).push(
+                            MaterialPageRoute(
+                              builder: (context) =>
+                                  const DepositPage(rfqState: RfqState()),
+                            ),
+                          );
+                        },
+                        child: Text(Loc.of(context).deposit),
+                      ),
```

Comment:
hmm would this make more sense to pull out of `_buildAccountBalance`?

Reply:
yep, can make that happen